.github/workflows/benchmarks_pr.yml

```diff
@@ -0,0 +1,123 @@
+name: Benchmarks
```

Comment:
Maybe Continuous Benchmarks to match the other workflow?

Reply:
I thought they would conflict

Reply:
no you will just see the branch appear (main or the one from the PR)